src/test/java/lotto/domain/LottoFixture.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoFixture {
+    public static Lotto lottoFixture(String manualInput) {
+        return new Lotto(lottoNumbersFixture(manualInput));
+    }
+
+    public static WinLotto winLottoFixture(String input) {
+        return new WinLotto(lottoNumbersFixture(input));
+    }
+
+    public static WinLotto winLottoWithBonusFixture(String input, LottoNo bonusNumber) {
+        return new WinLotto(lottoNumbersFixture(input), bonusNumber);
+    }
+
+    public static LottoNumbers lottoNumbersFixture(String manualInput) {
```

Comment:
```suggestion
    public static LottoNumbers lottoNumbersFixture(int... numbers) {
```
테스트 하기 위해 매번 문자열로 감쌀 필요가 없을 것 같아요!
숫자를 받아서 바로 처리하면 될 것 같습니다